src/ServiceControl.Audit/Auditing/AuditIngestion.cs

```diff
@@ -180,7 +180,7 @@ async Task EnsureStopped(CancellationToken cancellationToken)
             try
             {
                 await startStopSemaphore.WaitAsync(cancellationToken);
-                await StopAndTeardownInfrastructure(cancellationToken);
+                await StopAndTeardownInfrastructure(new CancellationToken(canceled: true));
```

Comment:
@danielmarbach what do you think about this? 

This should result in terminating ASAP but still release any resources.

I'm not sure if transports would properly teardown on stop but if not that should be considered a but right?

Reply:
Is the idea here to make sure no transports waits for pending receive operations?

Reply:
Yes

Reply:
> Is the idea here to make sure no transports waits for pending receive operations?

@danielmarbach yes, but ideally still correctly teardown any resources (because we mix both in Stop..)

Reply:
Added code comments to clarify why the cancellationtoken is passed in the cancelled state